client/src/hooks/useSubscribeToMore.ts

```diff
@@ -0,0 +1,11 @@
+import { useEffect } from 'react';
+
+export const useSubscribeToMore: any = ({ options, subscribeToMore } : { options: any, subscribeToMore: Function }) => {
+
+  useEffect(()=>{
+    Object.keys(options).forEach(key => {
```

Comment:
Any strong reason why we pass these in as an object? Could we not pass them as a list?